packages/dbos-cloud/cli.ts

```diff
@@ -139,6 +140,22 @@ userdbCommands
     process.exit(exitCode);
   }))
 
+/////////////////////
+/* USER DASHBOARDS */
+/////////////////////
+
+const dashboardCommands = program
+  .command('dashboard')
+  .description('Manage Monitoring Dashboards')
+
+dashboardCommands
+  .command('init')
```

Comment:
I think it's better to call it `launch` because the dashboard only needs to be initialized once, but can be launched multiple times. Wdyt?

Reply:
Made the change